libcrux_crypto/src/rand.rs

```diff
@@ -1,11 +1,13 @@
 use std::sync::RwLock;
 
-use libcrux::drbg::Drbg;
 use openmls_traits::random::OpenMlsRand;
 
-/// The libcrux-backed randomness provider for OpenMLS
+use rand::{rngs::OsRng, rngs::ReseedingRng, TryRngCore};
+use rand_chacha::ChaCha20Core;
+
+/// The randomness provider for the libcrux-backed OpenMLS provider
 pub struct RandProvider {
```

Comment:
Shall we drop this one? We could just implement `OpenMlsRand` for the crypto provider and would avoid duplicating code here.
I know that this was the style before. But I think that was more due to historic reasons, not because it made sense.